src/Config/SessionConfig.php

```diff
@@ -246,6 +263,27 @@ public function setHashBitsPerCharacter($hashBitsPerCharacter)
     }
 
     /**
+     * Set session.sid_bits_per_character
+     *
+     * @param  int $sidBitsPerCharacter
+     * @return SessionConfig
+     * @throws Exception\InvalidArgumentException
+     */
+    public function setSidBitsPerCharacter($sidBitsPerCharacter)
+    {
+        if (!is_numeric($sidBitsPerCharacter)
+            || !in_array($sidBitsPerCharacter, $this->validSidBitsPerCharacters)
```

Comment:
`||` should be `&&` ? also, I think in_array can use strict compare with pass `true` in 3rd parameter:  

```php
!in_array($sidBitsPerCharacter, $this->validSidBitsPerCharacters)
```